### ISSUE TITLE
Cannot delete targeted image highlight (1C)

### ISSUE THREAD

**Issue by blms** [performant-software/dm-2#315]
When a highlight on an image is targeted by clicking on a link to the highlight, the highlight cannot be (visually) deleted.

https://user-images.githubusercontent.com/7234006/127158671-7ae8f2e5-20e0-46bf-8825-1a05443bdc33.mov

**Comment by blms** [comment]
@MartinFoys This turns out to be a tricky issue. 

Fabric.js, our image highlight tool, only allows us to use a single shadow for glow—which on its own looks a bit weak and hard to see. So in order to make the glow darker, I copied the highlight several times. This caused the bug we see here. 

(I tried grouping the copies together to act as one, but apparently you _also_ can't apply a shadow to multiple items in a group.)

I've opted for the following solution—when in "edit" mode and you selected a targeted highlight in an image for transformation or deletion, the glow disappears:


https://user-images.githubusercontent.com/7234006/127200552-b3c5fa0f-59e9-4b8f-afb7-63c4d48e3521.mov


Let me know if this is acceptable.

Another possible solution is to just use the weak glow:

<img width="236" alt="Screen Shot 2021-07-27 at 12 14 02 PM" src="https://user-images.githubusercontent.com/7234006/127190084-b17dcd52-658d-4cc9-bc12-eeea6a770fe1.png">

Getting the "glow" to properly
